packages/driver/src/cy/commands/local_storage.ts

```diff
@@ -3,6 +3,8 @@ import _ from 'lodash'
 import $errUtils from '../../cypress/error_utils'
```

Comment:
Can we rename this file from local_storage since this handles both local & session storage?

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/24859/commits/8437b8eca6a8ecfa0a3ebe447306c484a476f63e